test/PushNotificationsTests/APITests.cpp

```diff
@@ -29,9 +29,9 @@ namespace Test::PushNotifications
     private:
         wil::unique_event m_failed;
 
-        const std::wstring c_testPackageFile = g_deploymentDir + L"PushNotificationsTestPackage.msixbundle";
-        const std::wstring c_testPackageCertFile = g_deploymentDir + L"PushNotificationsTestPackage.cer";
-        const std::wstring c_testPackageFullName = L"PushNotificationsWin32App_1.0.0.0_x64__9eg3vg7cmy3rj";
+        const std::wstring c_testPackageFile = g_deploymentDir + L"MSIXPackager_1.0.8.0_x64.msixbundle";
```

Comment:
Do we have a dependency on the app version? If yes, should we change the value of the const strings every time we increase the app version number?

Reply:
Yeah. We need to. That is the way it works right now. But I think we can improvise this to look for a single version under the deploymentdir. That way we don't have to change the version number. Thanks.